docs/questions-and-answers.md

```diff
@@ -0,0 +1,43 @@
+TL;DR, OSTree commit in container image is meant only to deliver OS updates (OSTree commits), not bootable disk images. 
+
+### How much deduplication do we still get with this new approach?
+
+Unfortunately, today, we do indeed need to download more than actually needed, but the files will still be deduplicated on disk, just like before. So we still won't be storing extra files, but we will be downloading extra files.
+But for users doing offline mirroring, this shouldn't matter that much. In OpenShift, the entire image is downloaded today, as well. 
+Nevertheless, see https://github.com/ostreedev/ostree-rs-ext/#integrating-with-future-container-deltas.
+
+### Will there be support for "layers" in the OSTree commit in container image?
+
+Not yet, but, as mentioned above, this opens up the possibility of doing OCI style derivation, so this could certainly be added later. It would be useful to make this image as familiar to admins as possible. Right now, the ostree-rs-ext client is only parsing one layer of the container image.
+
+### OSTree commits from OSTree repos uses the concept of branches, where commits have parents, etc. Are we expecting to retain these Git-like characteristics and somehow make it work with container images?
+
+Probably not. Again, we would like to make OSTree-commit-in-container-image as familiar to admins as possible, and really try to spare admins from having to learn too much about OSTree (while still benefiting from the advantages of OSTree, of course). 
+We would replace the concept of branches with OCI repository names and tags that point to specific OSTree-commit-in-OCI-images. For example, instead of `fedora:fedora/x86_64/coreos/stable`, we'd want to use something like `quay.io/coreos/fcos:stable`. And in tools like rpm-ostree that will make use of these new ostree-rs-ext features, e.g. `rpm-ostree deploy` will take in a repository name + (tag or digest) AKA `docker-reference` instead of branches (revisions) or commit IDs as an argument; `rpm-ostree status` should display the image's repository(plus tag?) rather than the OSTree branch (would it also show the container image digest along with the OSTree commit ID?).
+(Would each stream have its own repository (e.g. `quay.io/coreos/fcos-stable`) or would we rely on tags and use `quay.io/coreos/fcos:stable`? If the latter, how would Zincati know which release stream it is on? I think the confusion here is because we don't have a perfectly analogous relationship between branches/commit and repositories/tags/digests)
```

Comment:
I do still have some questions that I haven't sorted out for myself in this section here. Perhaps these bits would be more fitting in an rpm-ostree issue instead of here?

Reply:
I think this sub-thread might be best captured in a sub-thread at https://github.com/coreos/fedora-coreos-tracker/issues/812 since it's currently FCOS specific (although I still hope other editions adopt zincati).

Reply:
Moved to https://github.com/coreos/fedora-coreos-tracker/issues/812#issuecomment-865073670 :)